lambeq/backend/drawing/drawing.py

```diff
@@ -100,6 +100,9 @@ def draw(diagram: Diagram, **params) -> None:
 
     params['asymmetry'] = params.get(
         'asymmetry', .25 * needs_asymmetry(diagram))
+    default_label_flag = not diagram.has_frames
+    params['draw_type_labels'] = params.get('draw_type_labels',
```

Comment:
@Ragunath1729 you can simplify this into:

```python
params['draw_type_labels'] = params.get('draw_type_labels',
                                        not diagram.has_frames)
```

Reply:
have made the changes, please check now